src/scripts/generate_covid_case_datamart.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import pathlib
+import json
+from typing import List, Callable
+from utils import read_blob, record_combination_func, write_blob
+
+
+def standardize_lab_result(result: str) -> str:
+    """
+    Given a covid test result and dictionary whose keys (POSITIVE, NEGATIVE, and
+    INCONCLUSIVE) represent standard results and whose values are lists containing
+    non-standard results that map to the associated standard result then return a
+    standardized result. Return null is no standard result could not be found.
+    """
+    standard = json.loads(
+        open(
+            pathlib.Path(__file__).parent / "assets" / "covid_lab_results_standard.json"
+        ).read()
+    )
```

Comment:
We could probably load the standards dict outside of this function, which seems to handle mapping a singular result (or small set of results). I would imagine the most appropriate place for this is right before a big iterator loop over a set of files / results. IO can be fast, but because the dictionary is so small I see no disadvantage to just keeping it in memory.

Reply:
Moved out of this function. See comment below.